app/views/home/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Hello World!</h1>
+<h1>Hello <%= @user.name %></h1>
+<% if @user.picture.attached? %>
+  <p>
+  <strong>Picture:</strong>
+  <br>
```

Comment:
Maybe this should be a `dl`/`dt`/`dd` instead? Does that more closely match the semantics?

Reply:
Ah, maybe. I am unfamiliar with <dl/dt/dd>. I was focused only on functionality, but after looking into it, perhaps you're right!